templates/Bulk_RNA_seq/Templates/biojupies/base.j2

```diff
@@ -0,0 +1,35 @@
+{% extends "default/base.j2" %}
+{% block head %}
+<title>Bulk RNA-seq analysis pipeline</title>  
```

Comment:
If this is your only change to the `biojupies` profile, you should instead simply:

```
{% extends "biojupies/base.j2" %}
{% block head %}
<title>Bulk RNA-seq analysis pipeline</title> 
<link rel="shortcut icon" href="https://image.flaticon.com/icons/svg/65/65987.svg"/>
<link rel="icon" href="https://image.flaticon.com/icons/svg/65/65987.svg">  
{% endblock %}
```